content/v2.0/reference/cli/influx/pkg/export/all.md

```diff
@@ -17,11 +17,12 @@ influx pkg export all [flags]
 ```
 
 ## Flags
-| Flag           | Description                                                                     | Input Type | {{< cli/mapped >}} |
-|:----           |:-----------                                                                     |:---------- |:------------------ |
-| `-f`, `--file` | Package output file. Defaults to stdout. Use `.yml` or `.json` file extensions. | string     |                    |
-| `-h`, `--help` | Help for the `export` command                                                   |            |                    |
-| `-o`, `--org`  | The name of the organization that owns the resources                            | string     | `INFLUX_ORG`       |
-| `--org-id`     | The ID of the organization that owns the resources                              | string     | `INFLUX_ORG_ID`    |
+| Flag           | Description                                                                                   | Input Type      | {{< cli/mapped >}} |
+|:----           |:-----------                                                                                   |:----------      |:------------------ |
+| `-f`, `--file` | Package output file. Defaults to stdout. Use `.yml` or `.json` file extensions.               | string          |                    |
```

Comment:
Does it make sense to say "Template file/Package template...file"?
...if package exports only include templates/template resources?